lib/bot.js

```diff
@@ -144,9 +144,18 @@ class Bot {
         this.ircClient.say(ircChannel, prelude);
         this.ircClient.say(ircChannel, text);
       } else {
-        text = `<${coloredUsername}> ${text}`;
-        logger.debug('Sending message to IRC', ircChannel, text);
-        this.ircClient.say(ircChannel, text);
+        if (text !== '') {
+          text = `<${coloredUsername}> ${text}`;
+          logger.debug('Sending message to IRC', ircChannel, text);
+          this.ircClient.say(ircChannel, text);
+        }
+        if (message.attachments) {
```

Comment:
Can a message have both an attachment and a message? If not, maybe it would be better to do 

```js
if (message.attachments) { 
  // something
} else {
  // send regular message
}